dist/vue-datepicker.min.js

```diff
@@ -1 +1 @@
cker-day.between[data-v-dfec4766]:not(.disabled),.datepicker-days .datepicker-day.in-range[data-v-dfec4766]:not(.disabled){color:#fff}.datepicker-days .datepicker-day.between:not(.disabled) .datepicker-day__effect[data-v-dfec4766],.datepicker-days .datepicker-day.in-range:not(.disabled) .datepicker-day__effect[data-v-dfec4766]{transform:translateX(-50%);opacity:1;left:0;width:calc(100% + 1px);border-radius:0;opacity:.5}.datepicker-days .datepicker-day.between:not(.disabled) .datepicker-day__effect[data-v-dfec4766]:before,.datepicker-days .datepicker-day.in-range:not(.disabled) .datepicker-day__effect[data-v-dfec4766]:before{opacity:1;left:50%}.datepicker-days .datepicker-day.selected[data-v-dfec4766]{color:#fff}.datepicker-days .datepicker-day.selected:hover:not(.disabled) .datepicker-day__effect[data-v-dfec4766]{opacity:1}.datepicker-days .datepicker-day.selected .datepicker-day__effect[data-v-dfec4766]{transform:translateX(-50%);opacity:1}.datepicker-days .datepicker-day.first .datepicker-day__effect[data-v-dfec4766],.datepicker-days .datepicker-day.select-start:hover:not(.selected) .datepicker-day__effect[data-v-dfec4766]{opacity:1}.datepicker-days .datepicker-day.first .datepicker-day__effect[data-v-dfec4766]:before,.datepicker-days .datepicker-day.select-start:hover:not(.selected) .datepicker-day__effect[data-v-dfec4766]:before{opacity:.5;left:50%}.datepicker-days .datepicker-day.last .datepicker-day__effect[data-v-dfec4766],.datepicker-days .datepicker-day.select-end:hover:not(.selected) .datepicker-day__effect[data-v-dfec4766]{opacity:1}.datepicker-days .datepicker-day.last .datepicker-day__effect[data-v-dfec4766]:before,.datepicker-days .datepicker-day.select-end:hover:not(.selected) .datepicker-day__effect[data-v-dfec4766]:before{opacity:.5;left:-50%}.datepicker-days .datepicker-day.first.last .datepicker-day__effect[data-v-dfec4766]:before{opacity:0}.datepicker-days .datepicker-day.disabled[data-v-dfec4766]{cursor:default;color:rgba(0,0,0,.26)}.datepicker-days .datepicker-day.disabled:hover .datepicker-day__effect[data-v-dfec4766],.datepicker-days .datepicker-day.disabled:hover .datepicker-day__effect[data-v-dfec4766]:before{opacity:0!important}.datepicker-days .datepicker-day--current[data-v-dfec4766]{position:absolute;top:1px;left:50%;transform:translateX(-50%);width:34px;height:34px;border-radius:50%;border:1px solid currentColor}@media only screen and (min-width:768px){.datepicker-days .datepicker-day--current[data-v-dfec4766]{top:4px;width:36px;height:36px}}.datepicker-days .datepicker-day__effect[data-v-dfec4766]{position:absolute;top:1px;left:50%;width:34px;height:34px;border-radius:50%;transition:all 450ms cubic-bezier(.23,1,.32,1);transition-property:transform,opacity;transform:translateX(-50%) scale(0)}@media only screen and (min-width:768px){.datepicker-days .datepicker-day__effect[data-v-dfec4766]{top:4px;width:36px;height:36px}}.datepicker--range .datepicker-days .datepicker-day__effect[data-v-dfec4766]:before{content:"";po
```

Comment:
Did you start command yarn build ? Should have 3 build files.

Reply:
Yes I did. I wonder why there are missing files 🤔